doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md

```diff
@@ -2,6 +2,11 @@
 
 Set 2 videos are not available yet. Check back on or before the 11th of October 2022.
 
+## Tasks
+{% embed url="[https://youtu.be/IhpzEYTue34](https://youtu.be/IhpzEYTue34)" %}
+Task for Part 1 set 2
+{% endembed %}
+
```

Comment:
Nice try @Eddayavuz 
But this will not show up properly on gitbook, but you got the idea.
Please see embed examples in this file: https://github.com/Ifycode/git-github-training-resource/blob/develop/doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md
This should help you get the embed right.

Reply:
Instructions about what your pull request title should look like can be found here: https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/making-and-submitting-changes#contribution-guidelines
Please change that too so that it's easy to track your pull request.

Reply:
Hello, @Ifycode I did the necessary changes. Hope it works!

Reply:
Nicely done @Eddayavuz
Yes the embed is correct now.

Remove the `#47` in your pull request title. Once you have done that, I will approve the pull request.

Reply:
I removed it :)

Reply:
Thank you. Approved!